GDevelop.js/Bindings/Bindings.idl

```diff
@@ -54,6 +54,11 @@ interface VectorObjectFolderOrObject {
     [Const] ObjectFolderOrObject at(unsigned long index);
 };
 
+interface VectorScreenshot {
+    unsigned long size();
+    [Value] Screenshot at(unsigned long index);
```

Comment:
Why not?

```suggestion
    [Const] Screenshot at(unsigned long index);
```

Returning a [Value] is risky because it uses a unique static variable. This means that doing:

const myScreenshot1 = myVector.at(0);
const myScreenshot2 = myVector.at(1);

//myScreenshot1 is now the SAME as myScreenshot2, because they both point to a unique, static object in C++ memory that is overriden every time it's called.

This also goes against what other VectorXXX are doing.

[Value] is almost a hack, if we could we would never use it.